fog/view/server/src/sharding_strategy.rs

```diff
@@ -48,6 +59,22 @@ impl EpochShardingStrategy {
     pub fn new(epoch_block_range: BlockRange) -> Self {
         Self { epoch_block_range }
     }
+
+    fn have_enough_blocks_been_processed(&self, processed_block_count: BlockCount) -> bool {
+        if self.is_first_epoch() {
+            return true;
+        }
+
+        let epoch_block_range_length =
+            self.epoch_block_range.end_block - self.epoch_block_range.start_block;
+        let minimum_processed_block_count = epoch_block_range_length / 2;
+
+        u64::from(processed_block_count) >= minimum_processed_block_count
+    }
+
+    fn is_first_epoch(&self) -> bool {
```

Comment:
Like the functionalization here.  Even though it's only one line, the function name more clearly communicates intent where it's used.